pos_report_session_summary/session_summary_report.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<odoo>
+    <data>
```

Comment:
Pleas remove `data` tag in compliance with https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-10.0#tasks-to-do-in-the-migration